src/Master.cairo

```diff
@@ -204,33 +202,50 @@ mod Master {
             self._total_contribution.read(month_id)      
         }
 
-        fn get_contributions_data(self: @ContractState, contributor: ContractAddress, guild: felt252) -> Array<u32> {
-            self._contributions_data.read((contributor, guild))      
-        }
-
-        fn get_dev_points(self: @ContractState, contributor: ContractAddress) -> u32 {
-            let contribution: Contribution = self._contributions.read(contributor);
-            contribution.dev  
-        }
-
-        fn get_design_points(self: @ContractState, contributor: ContractAddress) -> u32 {
-            let contribution = self._contributions.read(contributor);
-            contribution.design
-        }
-
-        fn get_problem_solving_points(self: @ContractState, contributor: ContractAddress) -> u32 {
-            let contribution = self._contributions.read(contributor);
-            contribution.problem_solving
+        fn get_guild_total_contribution(self: @ContractState, month_id: u32, guild: felt252) -> u32 {
```

Comment:
Question - how detailed should tests be? If very then we might want to test `get_guild_total_contribution` and `get_guild_points` functions for every guild in a specific test.